lib/xdrgen/generators/rust/src/types.rs

```diff
@@ -914,6 +916,55 @@ impl<T: schemars::JsonSchema, const MAX: u32> schemars::JsonSchema for VecM<T, M
     }
 }
 
+#[cfg(feature = "schemars")]
+impl<T, TA, const MAX: u32> serde_with::schemars_0_8::JsonSchemaAs<VecM<T, MAX>> for VecM<TA, MAX>
```

Comment:
This interface is required so that VecM<> can be referenced in serde_as attributes.